bundle/edu.gemini.pot/src/main/java/edu/gemini/pot/sp/memImpl/MemTemplateFolder.java

```diff
@@ -86,6 +86,7 @@ public void setTemplateGroups(List<? extends ISPTemplateGroup> newGroupList) thr
         List<ISPTemplateGroup> newCopy = new ArrayList<>(newGroupList);
         getProgramWriteLock();
         try {
+            SPAssert.setsNoDuplicateObs(this, newCopy);
```

Comment:
I guess this throws an exception, Will it be displayed to the user, and if so what can he do to solve the problem?